public/views/content/studentView.html

```diff
@@ -0,0 +1,3 @@
+<a ui-sref=".addEntry">+ Add new entry</a>
```

Comment:
I suppose I just don't understand what the problem is, but it'd be nice to resolve it sooner since not merging is making it challenging to make much progress